src/maker/config.rs

```diff
@@ -195,8 +195,8 @@ impl MakerConfig {
     }
 
     // Method to manually serialize the Maker Config into a TOML string
-    pub fn to_toml_string(&self) -> String {
-        format!(
+    pub fn update_maker_config(&self, file_path: &std::path::Path) {
```

Comment:
Call this `write_to_file`. Because we can use this even when writing fresh config to a file. Not always update.